scripts/install_rclone.bash

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#
+#  Installs the latest version of rclone plugin
+#
+
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
+set -o errexit   # abort on nonzero exitstatus
+set -o nounset   # abort on unbound variable
+set -o pipefail  # don't hide errors within pipes
+IFS=$'\n\t'
+
+
+R_CLONE_VERSION="1.58.0"
+curl --silent --location --remote-name "https://downloads.rclone.org/v${R_CLONE_VERSION}/rclone-v${R_CLONE_VERSION}-linux-amd64.deb"
```

Comment:
I have the impression that the binary in the repo actually was unintended, ... 

How did you manage to commit this? There is a pre-commit hook to avoid this scenario?

Reply:
No recommit hooks run on my machine by default. I think I have to restore the configurations.

Reply:
The pre-commit hooks are **extremely important** to have the working. If there are some unknown problems just let us know